packages/react/src/experimental/OneDataCollection/index.stories.tsx

```diff
@@ -256,9 +256,11 @@ const createPromiseDataFetch = (delay = 500) => {
 const ExampleComponent = ({
   useObservable = false,
   usePresets = false,
+  fixedCols = 0,
```

Comment:
I suggest to name it _frozenCols_ or _freezeCols_ because this is the term used in all data sheet processors like excel and google spreadsheets.

Fixed columns could be interpreted as "fixed width column" which brings ambiguity

Reply:
makes sense